zookeeper-server/src/main/resources/lib/commons-io-2.7.LICENSE.txt

```diff
@@ -0,0 +1,203 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```

Comment:
I am not sure we need this file

btw I like to have a consistent way of handling license files (having the license for every jar file is better)

Reply:
Right.  I added that file because we already have equivalents for `jetty`, `jline`, `log4j`, `metrics`, `netty`, `simpleclient`, `slf4j` and `snappy`.

Are you saying that the set should be completed all at once, rather than piecewise?  If so, would you like me to remove this one?